src/utils/DarCollectionUtils.js

```diff
@@ -16,7 +16,7 @@ import {
   concat,
   findIndex,
   cloneDeep,
-  groupBy
+  groupBy, flatten
```

Comment:
minor nitpick:
```suggestion
  groupBy,
  flatten
```
To just add a single newline chara here